django/test/runner.py

```diff
@@ -101,6 +101,43 @@ def debug(self, error):
         pdb.post_mortem(traceback)
 
 
+class TestCaseDTO:
+    """
+    The ParallelTestSuite requires that all test cases be pickleable. When a
+    SubTest fails with an exception, it's pickled and shipped back to the main
+    process to be displayed to the user. Unfortunately, Django's TestCases
+    aren't always pickleable (for example, they may contain an instance of
+    django.test.Client, which can't be cleanly pickled if an exception was
+    raised by the view).
+
+    Luckily the outer TestRunner only cares about a small subset of the fields
+    on TestCase, and those are usually pickleable. We use this object to pluck
+    out those fields, discarding extra unpickleable members.
+    """
+
+    def __init__(self, test):
+        self.__id = test.id()
```

Comment:
Please use at most one underscore, double underscores trigger name mangling which makes subclasses easier to use.